packages/trace-viewer/src/ui/networkTab.tsx

```diff
@@ -173,7 +182,10 @@ function sort(resources: Entry[], filter: Filter | undefined, negate: boolean) {
 }
 
 function comparator(filter: Filter | undefined) {
-  if (filter === 'time')
+  if (filter === 'start')
+    return (a: Entry, b: Entry) => a._startedDateTime - b._startedDateTime;
```

Comment:
- This needs some extra support for old traces.
- Why not sort by `Entry._monotonicTime` instead, and drop `_startedDateTime`?